docs/optimization-framework/ab-testing.md

```diff
@@ -35,20 +31,24 @@ It is enabled by default but can be disabled via the configuration file:
 }
 ```
 
-Within the post edit screen click on the A/B icon to access the Experiments panel. Here you can set the alternative titles, the amount of traffic to show the tests to as well as start and end dates.
+Within the post edit screen click on the A/B icon to access the Experiments panel. Here you can set the alternative titles and/or alternative featured images, the amount of traffic to show the tests to as well as start and end dates.
 
 Once you are ready to run the test click on the toggle to unpause it. Results are updated every hour until a statistically significant winner is found.
 
 ![A/B Testing Titles user interface](../assets/ab-tests-titles.png)
 
-By default post title A/B tests are enabled for Posts and Pages however custom post types can be supported using the `altis.experiments.titles` key either when registering the post type or in the `init` action, for example:
+By default post title and featured image A/B tests are enabled for Posts and Pages however custom post types can be supported using the `altis.experiments.titles` key either when registering the post type or in the `init` action, for example:
 
 ```php
 add_action( 'init', function () {
 	add_post_type_support( 'events', 'altis.experiments.titles' );
+	// and/or..
+	add_post_type_support( 'events', 'altis.experiments.featured_images' );
```

Comment:
Minor nitpick but I try to avoid using a slash like this in the public facing docs, plus "and/or" is saying "and or or"
```suggestion
By default post title and featured image A/B tests are enabled for Posts and Pages however custom post types can be supported using keys shown in the example below, either when registering the post type or on the `init` action, for example:

```php
add_action( 'init', function () {
	// Add support for title A/B tests to the 'events' post type.
	add_post_type_support( 'events', 'altis.experiments.titles' );

	// Add support for featured image A/B tests to the 'events' post type.
	add_post_type_support( 'events', 'altis.experiments.featured_images' );
```
```